starter-code/scripts/article.js

```diff
@@ -10,14 +10,17 @@ function Article (opts) {
 }
 
 Article.prototype.toHtml = function() {
+  var $source = $('#article-template').html();
+  var templateRender = Handlebars.compile($source);
+  return templateRender(this);
```

Comment:
So close! return statements always end function execution, so whats happening here is the placement of your return statment is preventing this.daysAgo and this.publishStatus from running.